packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -1178,29 +1180,41 @@ class FlutterPlugin implements Plugin<Project> {
         } // end def addFlutterDeps
 
         if (isFlutterAppProject()) {
+
             project.android.applicationVariants.all { variant ->
                 Task assembleTask = getAssembleTask(variant)
                 if (!shouldConfigureFlutterTask(assembleTask)) {
                   return
                 }
                 Task copyFlutterAssetsTask = addFlutterDeps(variant)
                 copyFlutterAssetsTask.doLast {
+
+                  def content = "";
+                  def outputDir = copyFlutterAssetsTask.destinationDir
+                  def publicKeyFile = new File("${project.rootDir.absolutePath}/../public.pem");
+
+                  def shorebirdYamlFile = new File("${outputDir}/flutter_assets/shorebird.yaml")
+                  if (publicKeyFile.exists()) {
+                    def base64EncodedKey = Base64.getEncoder().encode(publicKeyFile.readBytes());
+                    content += 'patch_public_key: ' + new String(base64EncodedKey) + '\n';
+                  }
```

Comment:
I think the public key just ends up as a flutter asset?  Or is the idea to include the content of the key in the shorebird.yaml itself?

Reply:
The later, to include the key (as base64) into the "final" shorebird.yaml